### ISSUE TITLE
Make pylint behave the same on GitHub and locally

### ISSUE THREAD

**Issue by kristinagalik** [mild-blue/txmatching#814]
V `txmatching/web/__init__.py`, vo funkcii create_app sa pylint stazuje ze je 51 statementov, pricom max-statements je setnuty na 50. Pylint sa vsak stazuje iba lokalne, ale nie na GitHube, aj ked na oboch miestach je setnuty rovnako.

**Comment by kristinagalik** [comment]
@kubantjan 
i tried using `--rcfile=.pylintrc` flag like so:
```
          pylint txmatching --rcfile=.pylintrc
          pylint local_testing_utilities --rcfile=.pylintrc
```
and github seems to find this file (because when i tried to allocate the file somewhere else, like `../.pylintrc` it reported that there is no such file) but chooses to ignore or maybe override the configuration? 
the only instance that i found on which it fails (when i didnt make any changes yet besides the flag) was when both directories txmatching and local_testing_utilities had score under 10.0 and it fails with exit code 8. so my first thought was maybe it takes into consideration only errors from the last command from the job? so i divided linter job into two jobs, one for each directory, just to make sure i understand the issue, and even when i solved linter issues for lets say txmatching, when i used the `--rcfile` flag, it again failed with exit code 8 even though the code was rated 10/10. (when its rated <10 the exit code changes to 26)
so it probably finishes the job and then writes the error even though the error didnt occur during execution of the last command.

**Comment by kristinagalik** [comment]
@jogobeny yes, i think so

**Comment by krllstdn** [comment]
Yep, sorry, I forgot to write it here.
When I worked on some task, there was an inconsistency between what I had locally and what was in github pipelines. Locally after running `make check` there were no errors, but in GH actions there was an error. When I tried to replicate it by literally copying the same line that caused the error in pipelines before (#1085), but wasn't noticed by local pylint, It now was considered as an error. So false alarm :)
